agentops/client.py

```diff
@@ -300,6 +307,12 @@ async def _record_event_async(self, func, event_name, *args, **kwargs):
                 returns = list(returns)
 
             event.returns = returns
+
+            # NOTE: Will likely remove in future since this is tightly coupled. Adding it to see how useful we find it for now
+            # TODO: check if screenshot is the url string we expect it to be? And not e.g. "True"
```

Comment:
Screenshots not being useful? Strikes me as something pretty valuable. Though perhaps we should make it "image" instead

Reply:
I'll remove this comment. I meant this feels pretty specific to multion and idk if we wanted that in our core code. But doesn't matter. Was not at all about removing screenshots, just the automatic assignment of screenshot from the returns

Automatically assigning screenshot from returns.screenshot to ActionEvent.screenshot for convenience for partners (multion)